homework9/src/components/Button/Button.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+import "./Button.css";
+
+function Button(props) {
+  return (
+    <Link to="/apply" className="link">
```

Comment:
for that component you can add possibility to change path `to={path}` 
Path can be not required props but with default value
`const {path="/apply" } = props`